openfisca_core/tests/test_reforms.py

```diff
@@ -101,8 +101,8 @@ def apply(self):
         )
     simulation = scenario.new_simulation(reference = True)
     reform_simulation = scenario.new_simulation()
-    assert str(simulation.calculate('birth')[0]) == '1980-01-01'
-    assert str(reform_simulation.calculate('birth')[0]) == '1970-01-01'
+    assert str(simulation.calculate('birth', None)[0]) == '1980-01-01'
+    assert str(reform_simulation.calculate('birth', None)[0]) == '1970-01-01'
```

Comment:
This not great, but adding a specific treatment for permanent variables is a lot of complexity, given the amount of code repetition 🤔 .

There are not so many permanent columns, so it's good enough for me now.